services/cell_logger/cell_logger.py

```diff
@@ -82,38 +81,47 @@ def write_to_db(timestamp, signal, cell_tech):
 
     print("Finished inserting cell signal power data for timestamp ", timestamp)
 
+#Read enviroment variables. If no env. variable is set, assume CSV logging. 
+
+default_value = 'CSV'
+log_env = os.getenv('CELL_LOG', default_value)
 
 #Create a metric to track cell signal
 
 signal_gauge = Gauge('avena_cell_signal_power', 'Received cell signal power')
 
-#Initialize postgres database
+#Initialize postgres database if "CELL_LOG" environment variable is set to log
+#to database.
+
+if((log_env == 'DB') or (log_env == 'DB,CSV') or (log_env == 'CSV,DB')):
```

Comment:
This seems fragile, but working. I say if we extend this anymore, we should look for a different solution.

Reply:
I agree. Something better could be done. Better to fix now than later in a rush.